terraform/modules/google-cloud/sql/variables.tf

```diff
@@ -28,7 +28,7 @@ variable "database_name" {
 
 variable "database_version" {
   description = "Version of the Cloud SQL database"
-  default     = "POSTGRES_15"
+  default     = "POSTGRES_17"
```

Comment:
Should we change this in our `environments` repo first before making it the default?